_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,207 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.4.9: # Images of Text (No Exception) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+    - Times Square image released into the public domain by (WT-shared) Ypsilonatshared at wts wikivoyage.
+---
+
+## Applicability
+
+This rule applies to any [visible][] [embedded image][] that is not [marked as decorative][].
+
+## Expectation
+
+For each of the image resources referenced by the [image sources][] of the test target, at least one of the following is true:
+
+- the image resource does not contain text expressing anything in a [human language][]; or
+- the image resource contains text and the text is not a significant part of the visual content in the image; or
+- the image resource contains text and it is [essential][] that the text is rendered with that specific presentation.
+
+## Assumptions
+
+- The text in the image resource is visible and not hidden by other resources rendered on the web page. If the text is not visible, even though parts of the image are, this rule may fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might by satisfied.
```

Comment:
I don't understand the rationale for this assumption. Why would such a situation fail 1.4.5 but not 1.4.9?

Reply:
the **rule** would fail. Both SCs would be satisfied.